apps/right-to-rent-check/translations/src/en/fields.json

```diff
@@ -117,7 +117,6 @@
   },
   "agent-email-address": {
     "label": "Email address",
-    "hint": "We will use this to let you know the outcome of the check",
```

Comment:
does this relate to a ticket?

Reply:
I'd have assume LCS-353, but then the content changes here don't match what's in the ticket. @sulthan-ahmed?

Reply:
yep that's the ticket https://jira.digital.homeoffice.gov.uk/browse/LCS-353

Most of the content is actually there, I just had to remove this hint and make the company field optional.  I've now updated my PR & commit with the ticket number.  I've also included a screenshot.  I hope that helps?  I can ask @Nadiahuq to review as it now has the screenshot

Reply:
Reviewed and happy. Thanks @sulthan-ahmed

Reply:
Thanks @Nadiahuq